### ISSUE TITLE
Remove cpu_features dependency

### ISSUE THREAD

**Issue by WeeJeWel** [mscdex/ssh2#1083]
The `cpu_features` dependency does not install on M1 Macs.

We use `homey -> dockerode -> docker-modem -> ssh2 -> cpu_features` in our module, which is installed using `npm i -g homey`, as usual.

Unfortunately, all M1 Mac users get a big error and they think the install failed.

**Comment by mikl** [comment]
@PRR24 maybe try apologising instead. Calling someone’s open source work “undermaintained” is downright rude. The words “With no disrespect” are not a magic spell that makes being rude ok.

@WeeJeWel cpu-features is included for good reasons. The error message is only a problem if someone doesn’t read the all-caps `SKIPPING OPTIONAL DEPENDENCY` npm helpfully prints. And as mscdex pointed out, this is just a temporary inconvenience that will be solved once the upstream library adds support for your fancy new Mac.

**Comment by diamondap** [comment]
Failure to build the cpu_features dependency can cause CI builds to fail, even though npm knows the dependency is optional. If you're using electron-build, turning off "build dependencies from source" is an effective temporary workaround. In package.json:

```
    "build": {
		...
        "buildDependenciesFromSource": false,
        ...
    },
```

**Comment by bmcilw1** [comment]
This is an issue for me because when I try to create a lambda it fails due to this package having unexpected syntax.

See what happens if you create a project, add this package as a dependency, create a serverless.yml file, and run `sls deploy`. It breaks with Unexpected Syntax due to this package.

**Comment by mscdex** [comment]
@bmcilw1 I don't see the connection between a vague "unexpected syntax" error and `ssh2`'s dependency on `cpu-features`

**Comment by bmcilw1** [comment]
I didn't specify, but I can recreate it shortly.

**Comment by worksofliam** [comment]
@priceaj Looks like other people have the same issues with ssh2 that we have.

**Comment by mscdex** [comment]
For those who are willing to test a new prebuilt addon system for me that may help alleviate some of the concerns raised (namely needing to have a build environment available for most common platforms), see https://github.com/mscdex/ssh2/issues/1114.

**Comment by technicalpickles** [comment]
The issue in cpu_features was fixed in 0.7.0 just released: https://github.com/mscdex/cpu-features/issues/7

**Comment by technicalpickles** [comment]
1.8.0 which includes https://github.com/mscdex/cpu-features/issues/7 was released, so this can be closed now: https://github.com/mscdex/ssh2/releases/tag/v1.8.0
